index.html

```diff
@@ -129,6 +129,13 @@ <h4>Student Support</h4>
                     </li>
                   </ul>
                 </li>
+                <li><a href="https://github.com/joeybish" target="_blank" rel="noopener noreferrer">Joseph Bishop</a>
+                  <ul>
+                    <li>
+                      Majors: Ecology & Evolutionary Biology and Optical Sciences & Engineering
```

Comment:
@tweissaz do we care about "&" vs &amp;? I'm not sure what is best practice these days. I also see "&" in other parts of the index.html file.

Reply:
I think this should be fine, as we're using it within the rest of the document and it's a pretty standard character at this point.